content/04.methods.md

```diff
@@ -169,6 +169,32 @@ All merged `SingleCellExperiment` objects were converted to `AnnData` objects an
 If a merged `SingleCellExperiment` object contained any ADT data, the RNA and ADT data were exported and saved separately as RNA (`_rna.h5ad`) and ADT (`_adt.h5ad`) objects.
 In contrast, if a merged `SingleCellExperiment` object contained HTO data due to the presence of any multiplexed libraries in the merged object, the HTO data was removed from the `SingleCellExperiment` object and not included in the exported `AnnData` object.
 
+### Analysis of bulk RNA-seq data
+
+#### Data preparation
+
+We identified solid tumor samples with both bulk and single-cell (or single-nuclei) RNA-seq data in the ScPCA Portal for analysis, with multiplexed samples excluded (N=105).
+We further identified low-quality samples based on visual inspection of quality control reports and removed (N=8), leaving a total of 97 samples across five ScPCA projects for analysis.
+
+For each project, we transformed and normalized with respect to library size bulk counts matrices for all samples using `DESeq2::rlog()` [@doi:10.1186/s13059-014-0550-8].
```

Comment:
I don't know it this goes too far on simplification, but the "with respect to library size" phrase just felt really awkward. And I think perhaps saying that we used `rlog` is sufficient to say what we were doing?
```suggestion
For each project, we transformed and normalized bulk counts matrices for all samples using `DESeq2::rlog()` [@doi:10.1186/s13059-014-0550-8].
```

Reply:
I don't think this goes too far, especially since we have open code for this